xml/System/String.xml

```diff
@@ -6628,7 +6635,7 @@ The `GetPinnableReference` method returns a character that can be used for pinni
  The following example searches for the "n" in "animal". Because string indexes begin at zero rather than one, the <xref:System.String.IndexOf%28System.String%29> method indicates that the "n" is at position 1.  
   
  [!code-cpp[System.String.IndexOf#12](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.String.IndexOf/CPP/simple1.cpp#12)]
- [!code-csharp[System.String.IndexOf#12](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.IndexOf/CS/simple1.cs#12)]
+ [!code-csharp-interactive[System.String.IndexOf#12](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.IndexOf/CS/simple1.cs#12)]
```

Comment:
the note is not being displayed before this example on the page. The note should be added to the remarks of the MemberGroup element to show up high in the page.

Reply:
Done